plan/planbuilder.go

```diff
@@ -325,6 +329,88 @@ func (b *planBuilder) buildAdmin(as *ast.AdminStmt) Plan {
 	return p
 }
 
+func detachIndexColumns(tn *ast.TableName) (indexOffset []int, columnOffset []int) {
```

Comment:
It's wrong to detach index columns. e.g. If the index is (a,b), Can you build histogram for b with the same algo used by building a? You should build a histogram for the tuple (a, b) !